### ISSUE TITLE
Need volunteers for Pilot test - May 11-15

### ISSUE THREAD

**Issue by zcorpan** [w3c/aria-at#162]
We need 6 to 12 volunteers to run the pilot test the week of May 11-15. The goal is to run through 3 test plans and produce test results, so we know that it works before we start the first complete test cycle.

If you are able to run tests (i.e., you know how to use the relevant screen reader), it would be great if you could contribute time that week for this. Please edit this comment to add your github username, or comment below. :-)

- [ ] JAWS/Chrome - 
- [ ] JAWS/Firefox - 
- [ ] NVDA/Chrome - 
- [ ] NVDA/Firefox - 
- [ ] macOS VoiceOver/Chrome - 
- [ ] macOS VoiceOver/Safari -

**Comment by jscholes** [comment]
@zcorpan Happy to test with NVDA/Chrome and NVDA/Firefox.

**Comment by jha11y** [comment]
@zcorpan I am happy to yes any combinations

**Comment by robfentress** [comment]
I'm happy to help with any of those test combinations, with the qualification that I'm still resolving some issues with our license of JAWS 2019, so I may be stuck with JAWS 2018.  Which versions of the AT and browsers are we using?  For macOS, we should also specify the OS version, since the VoiceOver is integrated into the OS.  I currently have access to machines with High Sierra and Catalina.

**Comment by zcorpan** [comment]
@robfentress good question! Let's discuss the issue of versions in #116

**Comment by jha11y** [comment]
I cannot edit the initial comment. My guess is because I'm not a member of this GitHub repo. Please let me know which combinations I should test

**Comment by zcorpan** [comment]
@robfentress I've added you to macOS VoiceOver/Chrome and macOS VoiceOver/Safari.

@jha11y I've added you to JAWS/Chrome and JAWS/Firefox.

Thanks!

**Comment by jscholes** [comment]
@zcorpan I'm also available to test with JAWS 2020 and Firefox/Chrome (in addition to NVDA as already indicated) if it would help.

**Comment by robfentress** [comment]
> @robfentress I've added you to macOS VoiceOver/Chrome and macOS VoiceOver/Safari.

So that means macOS 10.15.4, Chrome 81.0.4044.138, and Safari 13.1 assuming no changes before May 11?

**Comment by zcorpan** [comment]
Scheduling update: the new runner isn't yet ready for this pilot test, so we need to move the date. Apologies for the late rescheduling.

The new plan is to have the pilot test start on **Wednesday May 20** and end on **Tuesday May 26**.

**Comment by JasonAment** [comment]
I’d be more than happy to do NVDA or Voiceover. Please put me down for the set that has the greatest need.

**Comment by shirsha** [comment]
I can help with NVDA/Firefox and NVDA/Chrome

**Comment by zcorpan** [comment]
I've updated assignees and added instructions in the first post above.

**Comment by accessaces** [comment]
hello: i can test with  NVDA and jaws in both Firefox  and chrome and i can test all the types thanks lucy

**Comment by zcorpan** [comment]
Scheduling update: All functionality needed for pilot test isn't ready yet. We need to move the pilot test date again, unfortunately.

The new plan is **Wednesday May 27** to **Tuesday June 2**.

Thanks for your patience, and apologies for rescheduling.

**Comment by shirsha** [comment]
Can you please let me know if the invite is already sent to join the github team?
I couldn't find it it in my mail.
I went ahead and tried to getting into staging environment via  http://ara.w3.org/ .
Selected the "Authorize bocoup" button and got message saying unable to load

**Comment by zcorpan** [comment]
@shirsha the team we use (for now) is here: https://github.com/orgs/bocoup/teams/aria-at-report-testers . You have been invited to this team. You should be able to see your invitation here:

https://github.com/orgs/bocoup/invitation

After this is accepted, "Sign in with GitHub" in http://ara.w3.org/ should work.

**Comment by shirsha** [comment]
@zcorpan Thankyou!
I am able to navigate to http://ara.w3.org/test-queue page. I will start working once the Pilot Test test cycle is initiated.

**Comment by zcorpan** [comment]
Note: A new NVDA version came out last week or so, please make sure to upgrade. See https://github.com/w3c/aria-at/issues/116#issuecomment-624673182

**Comment by zcorpan** [comment]
Note: new URL to staging test runner that should be used for the pilot test is http://aria-at-staging.w3.org/ 

If you've previously signed in, the sign in should still work with the new URL.

**Comment by a11ydoer** [comment]
@zcorpan I can test mac OS/VO

**Comment by shirsha** [comment]
I have performed the following steps:

- Made sure the version for NVDA is 2020.1, Chrome is 81.0.4 and Firefox is 76.0.
- Selected the AT from settings.
- Assigned my name to the appropriate test plans in both "Test" and "Test Staging”.
- Entered all the test results in iframe 

But when I select the "Next Test" button, I got a message saying "Test 1 has not been completed in full and your progress on this test won't be saved." I have entered all required fields. Can you please let me know what am I missing?
This is happening in both "Test" and "Test Staging"  test cycle
<img width="1102" alt="Test assign" src="https://user-images.githubusercontent.com/7432373/82741967-e3390a00-9d1d-11ea-9f0c-f4de550c2a3d.png">
<img width="892" alt="Test-notcomplete" src="https://user-images.githubusercontent.com/7432373/82741968-e633fa80-9d1d-11ea-8ed7-3b99c4418049.png">

**Comment by zcorpan** [comment]
@a11ydoer I've invited you now and assigned you in the table above. Thanks!

**Comment by jesdaigle** [comment]
@shirsha Are you using the the "run as" button when you run tests?

**Comment by jha11y** [comment]
Can we test yet?  I haven't tried because the info because the emails say the tests would be ready starting tomorrow (May 27)

**Comment by s3ththompson** [comment]
@jha11y The pilot test has not officially started as there are still changes that need to be deployed to the app. Tomorrow is the official start of the test.

**Comment by shirsha** [comment]
I have few questions:

- Which Test  Cycle (Test or Test Staging) should I select? 
- I don't see run as option. Where can I find it?

**Comment by zcorpan** [comment]
In the CG call tomorrow we plan to walk through how to run tests. The first post in this thread also has instructions, we'll make sure they are correct tomorrow.

Here are [instructions for how to join the CG call](https://lists.w3.org/Archives/Public/public-aria-at/2020May/0011.html).

**Comment by zcorpan** [comment]
I believe @mcking65 wanted another test to land today before we start the pilot test. The plan is to start after the CG call today.

**Comment by zcorpan** [comment]
@JasonAment it looks like you still need to accept the invitation. You can do so here: https://github.com/orgs/bocoup/invitation

**Comment by zcorpan** [comment]
@shirsha 

> Which Test Cycle (Test or Test Staging) should I select?

"Test Pilot"

> I don't see run as option. Where can I find it?

After you've assigned yourself to a test plan, the test plan name in the left-most cell becomes a link. That link will start running the tests for that test plan.

See "Instructions for running pilot tests" above for more details; feel free to ask if something is still unclear.

**Comment by zcorpan** [comment]
In the CG call we talked briefly about redistributing the assignments, after combobox was removed. I haven't made any changes though. @mcking65 and @mfairchild365 can you follow up with this? I don't know what you wanted to test.

This could be done now in the app interface directly, and ignore the table above, if that is easier.

**Comment by JasonAment** [comment]
It tells me that the invite cannot be found.

Cheers,
Jason Ament

On May 27, 2020, at 11:28 AM, Simon Pieters <notifications@github.com> wrote:

﻿

@JasonAment<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FJasonAment&data=02%7C01%7C%7Ca4c804be17d04ca056d408d8026bce9a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637262009330040115&sdata=rxEJHvCgvZRyVgvJMmjVvICKnr%2FjJBUbX3tb1olGYPw%3D&reserved=0> it looks like you still need to accept the invitation. You can do so here: https://github.com/orgs/bocoup/invitation<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Forgs%2Fbocoup%2Finvitation&data=02%7C01%7C%7Ca4c804be17d04ca056d408d8026bce9a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637262009330050116&sdata=%2BYYm2Y8%2FixedRIjn4TcOTW02Nt5%2B4GvT08KKUtV5bsI%3D&reserved=0>

**Comment by zcorpan** [comment]
@JasonAment, hmm that's weird. I've invited you again now.

**Comment by mfairchild365** [comment]
> In the CG call we talked briefly about redistributing the assignments, after combobox was removed. I haven't made any changes though. @mcking65 and @mfairchild365 can you follow up with this? I don't know what you wanted to test.
> 
> This could be done now in the app interface directly, and ignore the table above, if that is easier.

Let's let people self-assign in the app interface. If you were assigned to the combobox test plan, or if you were not previously assigned to a test plan, or if you just have some extra time on your hands, feel free to self-assign to either the checkbox or menubar plans. We should have at least two testers for each combination of test plan + AT/browser.

If you are not sure how or where to assign yourself, comment here and we can help.

**Comment by zcorpan** [comment]
**Thank you** to everyone who contributed in the pilot test!

I look forward to hearing about this in the CG call today. What worked well, what could be improved. I see a lot of issues filed, many from @robfentress - thank you! I haven't yet reviewed them in detail, but it looks like we have some work to do.

Some tests are not completed. We can discuss whether to take the partial results or extend the pilot test.

**Comment by accessaces** [comment]
HELLO: UNFORTUNATELY I HAD TIME LAST WEEK BUT YOU SAID TO HOLD OFF AND I
WAS NOT ABLE TO GET TO THIS UNTILL TODAY IS THERE STILL A NEED FOR ME TO GO
IN AND TEST LUCY
lucia Greco
http://accessaces.com
follow me on twitter @accessaces

**Comment by zcorpan** [comment]
We discussed learnings from the pilot test in the CG call, minutes here: https://www.w3.org/2020/06/03-aria-at-minutes.html

The feedback is fantastic. We will go through everything and make a summary report.

We want to reach **complete test results by at least one tester for every test plan** for the pilot test. To that end, the last day of the pilot test is moved to **Sunday June 7**.

Please comment if you're assigned and have tests left but are not able to complete further tests this week.

Thanks again!

**Comment by jscholes** [comment]
@zcorpan Have completed all checkbox tests for JAWS/FF, JAWS/Chrome, NVDA/FF and NVDA/Chrome.  Let me know if there are any problems with something or I can provide further feedback about the UX as a whole.

**Comment by zcorpan** [comment]
Hi @accessaces! Thank you, I have invited you to the relevant GitHub team. Please see the top of this issue for instructions, and coordinate with @mfairchild365 and @mcking65 with assignments.

**Comment by robfentress** [comment]
I'll finish the menubar testing with VoiceOver using Chrome by the end of day on June 7th (probably sooner).
